core/modules/catalog/store/attribute/getters.ts

```diff
@@ -5,7 +5,11 @@ import RootState from '@vue-storefront/core/types/RootState'
 const getters: GetterTree<AttributeState, RootState> = {
   attributeListByCode: (state) => state.list_by_code,
```

Comment:
would be nice to change also these two to `getAattributeListByCode` and `getAttributeListById`

Reply:
even better - please consider to refactor state to not have list_by_id and by code but single list attributes. And then we have computed getters here which are returning these maps by id and code. We have a smaller initial state with this and no attributes redundancy.

Reply:
for now I've just changed that names because these state & getters are used in so many places